app/bundles/CoreBundle/Factory/MauticFactory.php

```diff
@@ -341,14 +347,20 @@ public function getLogger($system = false)
      */
     public function getHelper($helper)
     {
-        return match ($helper) {
-            'template.assets'     => $this->container->get('twig.helper.assets'),
-            'template.slots'      => $this->container->get('twig.helper.slots'),
-            'template.form'       => $this->container->get('twig.helper.form'),
-            'template.translator' => $this->container->get('twig.helper.translator'),
-            'template.router'     => $this->container->get('twig.helper.router'),
-            default               => $this->container->get('mautic.helper.'.$helper),
-        };
+        switch ($helper) {
```

Comment:
Why was this changed back from match to switch and the options are different?

Reply:
@escopecz The goal was to replace `container::get` (is deprecated), and to tighten services that are used in the `getHelper`. If i didn't miss any, then those are only ones that are used with the method.
I couldn't replace this with match, because on "default" we should warn users, at least in test case, otherwise rector should replace this with match.